app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
Tìm hiểu về `I18n in rails` và dùng I18n cho text `SampleApp`

Reply:
Qua Chap 3 em test